### ISSUE TITLE
Documentation for building / running

### ISSUE THREAD

**Issue by 8Keep** [Scrappers-glitch/SS-Editor#22]
I wish there were some documentation for building/running. Right now it seems to be using jdk packaged jfx which hasn't been included with the jdk for a long time.

**Comment by pavly-gerges** [comment]
Yeah, i am an old school guy who still use jdk-1.8, however, you can currently use jdk-10 without any problems, until adding the support for gradle 7+ and jdk-19.

**Comment by pavly-gerges** [comment]
There is a lot of boilerplate code inside the editor, sure i will need the community help, so making a wiki documentation for building and an architecture stuff would be great idea to put into considerations.
